server/services/interventionsService.test.ts

```diff
@@ -2846,6 +2846,59 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     })
   })
   */
+
+  describe('getServiceProviderReportingData', () => {
+    const reportingResponse = [
+      {
+        referralLink: 'https://refer-and-monitor.example/referral/c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        referralRef: 'SM1973AC',
+        referralId: 'c9f9e22f-ddd9-422a-a9af-76fc02b18b38',
+        contractId: 'thing',
+        organisationId: 'XYZ5678',
+        caseworkerId: 'liane.supplier@example.com',
+        serviceUserCrn: 'X017844',
+        dateReferralReceived: '2021-06-27T14:49:01+01:00',
```

Comment:
I'd expect a property whose name begins with `date` to be a date - i.e. no time components. Would it be more consistent to call these `referralReceivedAt`, etc? Or does this come down to the field names not being changeable in the CSV, and not wanting to do any mappings on the frontend?

Reply:
Yeah, for this I took all the headers Leigh supplied in the csv schema as I thought we could just use the fields as-is. However, you might have already seen in the next PR that we can actually map them to specific headers (I've just snake_cased everything), so I can update this object if you think it makes more sense? I guess I wanted to map them to the given schema as closely as possible.

Reply:
Yeah, I'm not too bothered either way. Since we're not doing anything on the frontend other than converting to CSV, we can just pass them through I think.